style-guide/styles/wds-variables/_buttons.scss

```diff
@@ -0,0 +1,3 @@
+$button-active-color: darken($color-link, 20%);
+$button-line-height: 1.3em;
+$hover-transition-duration: 300ms;
```

Comment:
Please rethink which variables should be exported from DS and mark rest as private variables of DS.